src/java/org/apache/cassandra/cql3/UpdateParameters.java

```diff
@@ -163,8 +166,21 @@ public Cell<?> addCell(ColumnMetadata column, ByteBuffer value) throws InvalidRe
 
     public Cell<?> addCell(ColumnMetadata column, CellPath path, ByteBuffer value) throws InvalidRequestException
     {
+        // General column value size
         Guardrails.columnValueSize.guard(value.remaining(), column.name.toString(), false, clientState);
 
+        if (column.type instanceof AsciiType) // Ascii size specific guardrail
+        {
+            Guardrails.columnAsciiValueSize.guard(value.remaining(), column.name.toString(), false, clientState);
+        } else if (column.type instanceof BytesType) // Blob size specific guardrail
```

Comment:
move to new line. There are a few other places that the `else if` is placed wrongly. Please correct them